scripts/config/ComposerSettings.php

```diff
@@ -1,46 +0,0 @@
-<?php
-
-global $wikiId;
-
-enableSemantics( "meza.$wikiId" );
```

Comment:
I'm not sure if we need this function call or not. The [SMW installation guide](https://github.com/SemanticMediaWiki/SemanticMediaWiki/blob/master/docs/INSTALL.md#step-5) says this is required. On MOD2, we had this in our LocalSettings.php. I haven't been able to find it in our config on FOD2. If it's truly not there, then maybe it's not required or maybe this is only required when initiating the SMW installation.

### Reference
* [SMW discussion page about possibly removing enableSemantics function call](https://semantic-mediawiki.org/wiki/Thread:User_talk:Kghbln/enableSemantics_no_longer_needed)
* [SMW Help:Installation](http://semanticmediawiki.org/wiki/Help:Installation#Troubleshooting)
* SemanticMediaWiki/SemanticMediaWiki#47 (see discussion)